src/utils/tests.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Router } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+import { render } from '@testing-library/react';
+import { AppContext } from '@edx/frontend-platform/react';
```

Comment:
Hmmmm.... this is going to be interesting when you port it to the admin portal. Presumably it has a different AppContext. We might want to rename and / or check what the admin portal does, because I think `AppContext` should probably live in it's app, not in the shared repo. If they're similar enough I could be wrong, though.

Reply:
good point!

Reply:
hmm @raq929 I don't see it being used at all! and now have this in both repos after refactor. Perhaps we can remove it..? I also did not find any usage of AppContext in admin portal repo

Reply:
will leave it here for now, removing from learner portal, the same function for now

Reply:
Admin portal does not use frontend-build or frontend-platform right now, so it does not utilize AppContext.